src/platform/polyfills/preESModulesPolyfills.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * This file contains polyfills for browsers that don't support
+ * the "nomodule" script attribute. These are generally older browsers
+ * like IE 11 that need more polyfills than the more common, newer versions
+ * of evergreen browsers
+ *
+ */
+
+// This section is for custom polyfills that we need
+import 'classlist-polyfill'; // DOM element classList support.
```

Comment:
IE 11 has some gaps in classList support, that's the only reason we still need it.